### DISCUSSION TITLE
Sharing images into markor

### DISCUSSION THREAD

**Discussion by harshad1** [gsantner/markor#2055] [Ideas]
Adding images to markor documents is a little hard.

In order to make this easier, we should be able to share images into markor.  The images should be  copied into the notebook directory if they are outside the notebook folder and a format-appropriate link should be added to the document they are added to.

i.e. share image -> select markor -> edit caption and path in the share into activity -> select existing or new document.

It will also be nice if we could somehow reduce the image size before copying it into markor.

**Comment by ghost** [comment]
I also think that sharing images would be amazing.

Regarding the second question: you can use `Insert Image -> Edit Picture`, and open it with your favourite image resing program.
